### PR TITLE
Sr main

### PR DESCRIPTION
**Pull request by andrew-rinato**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="theme-color" content="#ffffff">
+
+    <!-- Google Tag Manager -->
```
Comment (andrew-rinato) [inline public/index.html:15]: heres the 1/2 google trackers they wanted added

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-N2LKN6X');</script>
+    <!-- End Google Tag Manager -->
+
+    <title>Geese</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <!-- Google Tag Manager (noscript) -->
```
Comment (andrew-rinato) [inline public/index.html:28]: the other google tracker 2/2

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <div className ='title_div'>GEESE</div>
+            <Twiddle style={{height:'12px', width:'auto'}}/>
+              <a href="https://geese.lnk.to/cowboynudes" target="_blank" className= "link"> cowboy nudes </a>
+              <a href="https://www.youtube.com/watch?v=LvHsLdYXfaY" target="_blank" className= "link">watch</a>
+              <a href="https://www.bandsintown.com/a/342405-geese" target="_blank" className="link" >tour</a>
+              <a href="https://store.geeseband.com/" target="_blank" className="link" >merch</a>
+              <a href="http://eepurl.com/hkwcxv" target="_blank" className= "link">newsletter</a>
+          </div>
+        </div>
+        <div className='social_div'>
```
Comment (andrew-rinato) [inline src/index.js:33]: social media assets and overall site layout

File: src/index.css [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+  src: url('./assets/geeseband.otf');
+}
+
+@font-face {
+  font-family: 'geeseband';
+  src: url('./assets/Caslon Ionic-Regular Italic.otf');
+  font-style: italic;
+}
+
+.main_div {
```
Comment (andrew-rinato) [inline src/index.css:22]: in reality to get this to work as expected I arranged all the main components inside this main_div that is a flex box. html and body are also set to full height without margins

**Comment by nohr** [comment]
I've integrated the changes

**Pull request closed without merging**